FLEDGE.md

```diff
@@ -156,6 +159,8 @@ This will cause the browser to execute the appropriate bidding and auction logic
 
 The returned `auctionResultPromise` object is _opaque_: it is not possible for any code on the publisher page to inspect the winning ad or otherwise learn about its contents, but it can be passed to a Fenced Frame for rendering.  (The [Fenced Frame Opaque Source explainer](https://github.com/shivanigithub/fenced-frame/blob/master/OpaqueSrc.md) has initial thoughts about how this could be implemented.)  If the auction produces no winning ad, the return value can also be null, although this non-opaque return value leaks one bit of information to the surrounding page.  In this case, for example, the seller might choose to render a contextually-targeted ad.
 
+Optionally, `perBuyerTimeouts` can be specified to restrict the runtime (in milliseconds) of particular buyer's bidding scripts. If no value is specificed for a particular buyer, a default timeout of 100 ms will be selected. Any `perBuyerTimeouts` higher than 500 ms will be clamped to 500 ms.
```

Comment:
Hmm I guess this doesn't offer any way to change the _default_ timeout.  Maybe `'*'` could apply to any otherwise-unspecified buyer?

Reply:
Done.